app/views/step_by_step_pages/reorder.html.erb

```diff
@@ -26,31 +26,25 @@
     <%
       step_number = 1
       steps_length = @step_by_step_page.steps.length - 1
```

Comment:
This variable doesn't seem to be used anymore and looks like it can be removed.

Reply:
good spot - removed.